wordle-web/src/components/LetterBase.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <v-btn :style="setGradient()">
```

Comment:
The gradient to the buttons/tiles isn't bad and doesn't disrupt usability but it does feel a little clunky looking and could potentially be refined. Just so it looks a little smoother.